Description: use system shared libvpx library
 Qt WebEngine checks before build if system shared libvpx library works.
 .
 Previously the embedded code copy of Chromium depended on experimental
 parts of libvpx, but that was later discovered to be unneeded and code
 adapted since September 2016:
 <https://bugs.chromium.org/p/chromium/issues/detail?id=575651>
 .
 Apparently Qt WebEngine simply missed that change and didn't adapt its
 checks accordingly.
Author: Jonas Smedegaard <dr@jones.dk>
Last-Update: 2018-07-06

--- a/config.tests/libvpx/libvpx.cpp
+++ b/config.tests/libvpx/libvpx.cpp
@@ -26,7 +26,7 @@
 **
 ****************************************************************************/
 
-#include <vpx/svc_context.h>
+#include <vpx/vp8cx.h>
 #include <vpx/vpx_frame_buffer.h>
 #include <vpx/vp8dx.h>
 
